### ISSUE TITLE
Seems that pixmap-plist has been deleted from CLX

### ISSUE THREAD

**Issue by earl-ducaine** [sharplispers/clx#109]
The function `pixmap-plist` (and its setf) still have entries in the documentation, but 
```
(function xlib:pixmap-plist)
```
Returns an error (function undefined) when CLX is loaded.
Probably the fix would look something like adding the following lines to clx.lisp

```
(defun pixmap-plist (pixmap)
  (drawable-plist pixmap))

(defun (setf pixmap-plist) (value pixmap)
  (setf (drawable-plist pixmap) value))

```

**Comment by PuercoPop** [comment]
Indeed the fix you propose is the current strategy:

https://github.com/sharplispers/clx/blob/025527cc561caf86c920d5714075efaa61f31575/clx.lisp#L429-L434

I would propose we drop and `window-*` aliases. That functions referring to drawables apply to windows and pixmaps is already documented in the manual under [Section 5.1](https://sharplispers.github.io/clx/Drawables.html#Drawables):

> Both windows and pixmaps can be used as sources and destinations in graphics operations. These are collectively known as drawables. The following functions apply to both windows and pixmaps.

@dkochmanski Which alternative would you prefer?

**Comment by dkochmanski** [comment]
It could be also defined as an accessor in class definition with the same symbol (to assure that the same slot is used) instead of stray defuns.

```common-lisp
(defclass foo () ((a :accessor foo-a)))
(defclass bar (foo) ((a: accessor bar-a)))
```

I think that we can't drop them just like that. My preference would be removing them from the manual (or mark them as deprecated there), but keep the symbols for now. Then we'll wrap a release, after which we will mark them as obsolete.

**Comment by dkochmanski** [comment]
fixed by #111.
